### DISCUSSION TITLE
What about json-schema?

### DISCUSSION THREAD

**Discussion by uuf6429** [schemaorg/schemaorg#3430] [Q&A]
I don't know if this has been discussed before - unfortunately GH search turns up a tonne of false-positive results with json-schema. :/

I've been through the project a bit and found that the actual schema is described in about 6 formats (on the [developers page](https://webschemas.org/docs/developers.html) and [releases](https://github.com/schemaorg/schemaorg/tree/main/data/releases/24.0)) - none of those formats feel very useful (strictly in my opinion). Then there's the rest of the site, where the schema is being documented in HTML - which feels antiquated.

So the big question is, **why isn't json-ld schema available as json-schema?**
It feels natural, expected and more publicly useful.

A very limited version is already available here: https://json.schemastore.org/jsonld.json, but it lacks most of the schema.

**Why would this be useful?**
1. the various json-schema libraries out there could be useable with jsonld (e.g. data validation, form building etc)
2. it would allow easy integration within IDEs:
![image](https://github.com/schemaorg/schemaorg/assets/230049/747c89ee-037b-4d6b-935f-754b89b90336)
_(PhpStorm/IDEA Example - that stuff becomes available right after adding `"$schema": "https://json.schemastore.org/jsonld.json"` to the json object)_

**Comment by M-casado** [comment]
Any news about this, @uuf6429? Did you find a good and maintained repository of schema.org in JSON Schema?

**Reply by uuf6429** [reply]
no.. I also haven't worked in this area for a while either.

**Reply by M-casado** [reply]
Shame, but thanks for the update!
